From aaa124efa8d0d65f71663ee5a37710c7f5598d0f Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 24 Apr 2019 13:00:49 +0800
Subject: [PATCH 075/147] arm64: allwinner: h6: pine-h64: add PIO VCC bank
 supplies

The Allwinner H6 SoC features tweakable VCC for PC, PD, PG, PL and PM
banks.

This patch adds supplies for PC and PD banks. PG and PM banks are
used for Wi-Fi and should be added when Wi-Fi is added; PL bank is where
PMIC is attached, and currently if a PMIC regulator is added for it
dependency loop will happen.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
index 9e150b3a952d..65cb7167db30 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
@@ -139,6 +139,11 @@
 	status = "okay";
 };
 
+&pio {
+	vcc-pc-supply = <&reg_bldo2>;
+	vcc-pd-supply = <&reg_cldo1>;
+};
+
 &r_i2c {
 	status = "okay";
 
-- 
2.21.0

